virttest/vt_console.py

```diff
@@ -0,0 +1,145 @@
+import threading
+
+
+from . import remote
+
+
+class ConsoleError(Exception):
+    """A base class for console errors."""
+    pass
+
+
+class NoConsoleError(ConsoleError):
+    """Raise when there is no console available for a session."""
+
+    pass
+
+
+class ConsoleBusyError(ConsoleError):
+    """Raise when session tries to communicate with a console in use."""
+
+    pass
+
+
+def lock(function):
+    """
+    Get the ConsoleManager lock, run the function, then release the lock.
+
+    :param function: Function to wrap.
```

Comment:
doc string for 'return'

Reply:
It returns what `function` returns. I think we don't need that in doc string.